index.d.ts

```diff
@@ -308,8 +308,10 @@ declare module 'plaid' {
     category: Array<string> | null;
     category_id: string | null;
     date: Iso8601DateString;
+    authorized_date: Iso8601DateString;
```

Comment:
hey there, this field is nullable. mind updating? https://plaid.com/docs/#transaction-schema

Reply:
done